ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml

```diff
@@ -1518,34 +1485,37 @@ presubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
-    trigger: (?m)^/test( | .* )openshift-penultimate-operator-e2e-tests,?($|\s.*)
+    trigger: (?m)^/test( | .* )ocp-4-8-qa-e2e-tests,?($|\s.*)
   - agent: kubernetes
     always_run: false
     branches:
     - ^master$
     - ^master-
-    cluster: build02
-    context: ci/prow/openshift-penultimate-qa-e2e-tests
+    cluster: build03
```

Comment:
Unfortunately, `make update` keeps changing this and couple other `cluster:` attributes from `build02` to `build03`. All other `cluster`-s in this file are `build02`. Not sure if that would be an issue.

Reply:
it probably means OSCI automation is moving build clusters on us. If you rebase and stay in sync with master these changes should be OK. Otherwise I think CI will complain.

Reply:
It stays like this after the rebase so I guess it's fine.